release.nix

```diff
@@ -6,7 +6,7 @@ let
 
   pkgs = import <nixpkgs> {};
 
-  systems = [ "x86_64-linux" "i686-linux" "x86_64-darwin" /* "x86_64-freebsd" "i686-freebsd" */ ];
+  systems = [ "x86_64-linux" "i686-linux" "x86_64-darwin" "i686-cygwin" "x86_64-cygwin" /* "x86_64-freebsd" "i686-freebsd" */ ];
```

Comment:
The release attribute only determines the release job (http://hydra.nixos.org/job/nix/master/release). But this line:
```
    build = pkgs.lib.genAttrs systems (system:
```
causes Hydra to instantiate a job `build.x86_64-cygwin` that it won't be able to build.